spec/acceptance/ini_setting_spec.rb

```diff
@@ -23,20 +23,17 @@
 
     describe file(path) do
       it { should be_file }
-      #XXX Solaris 10 doesn't support multi-line grep
-      it("should contain #{content}", :unless => fact('osfamily') == 'Solaris') {
-        should contain(content)
-      }
+      its(:content) { should match content }
```

Comment:
yes, it works, I have tested it against solaris 10, it was the grep that was broken, its(:content) does not use the multiline grep so with a regex it matches.